HW__2.postman_collection.json

```diff
@@ -0,0 +1,238 @@
+							"\r",
+							"    console.log(_.difference(names1,names2))}\r",
+							"\r",
+							"//excluding_names([1,2,3,4],[2,56,2,6,7])"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "httpbin.org/get",
+					"host": [
+						"httpbin",
+						"org"
+					],
+					"path": [
+						"get"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "HW_2.2",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"const ajv = require('ajv');\r",
+							"const moment = require(\"moment\");\r",
+							"const uid = require(\"uuid\");\r",
+							"//const jsonBody = pm.response.json();\r",
+							"const ajvInstance = new ajv({logger: console});\r",
+							"const schema = {\r",
+							"    //\"type\": \"object\",\r",
+							"    //\"properties\": {\r",
+							"            \"date\": \"string\",\r",
+							"            \"guid\": \"string\",\r",
+							"            \"randomString\": \"string\",\r",
+							"\r",
+							"}//}\r",
+							"//console.log(typeof(moment().format('MMMM Do YYYY')));\r",
+							"//console.log({{$randomFirstName}});\r",
+							"const data1 = {\r",
+							"\"date\": moment().format('MMMM Do YYYY'),\r",
+							"\"guid\": uid(),\r",
+							"\"randomString\": moment() + uid(),\r",
+							"}\r",
+							"pm.test('valid', () => {\r",
+							"    pm.expect(ajvInstance.validate(schema, data1)).to.be.true;\r",
+							"});\r",
+							"//console.log(data1);\r",
+							"console.log(ajvInstance.errors);\r",
+							"\r",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://postman-echo.com/post",
+					"protocol": "https",
+					"host": [
+						"postman-echo",
+						"com"
+					],
+					"path": [
+						"post"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "HW_2.4",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"//pm.variables.set(\"nameUsers\", \"Annie\");\r",
+							"\r",
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "https://rickandmortyapi.com/api/character?id={{newID}}",
```

Comment:
Not correct path.

[Documentation](https://rickandmortyapi.com/documentation/#get-a-single-location): https://rickandmortyapi.com/api/character/{{id}}
id - is not query parameter.